packages/terra-search-field/src/terra-dev-site/doc/search-field/SearchField.1.doc.jsx

```diff
@@ -29,6 +29,8 @@ import SearchFieldFilterNumeric from '../example/SearchFieldFilterNumeric';
 import SearchFieldFilterNumericSrc from '!raw-loader!../../../../src/terra-dev-site/doc/example/SearchFieldFilterNumeric';
 import SearchFieldFocus from '../example/SearchFieldFocus';
 import SearchFieldFocusSrc from '!raw-loader!../../../../src/terra-dev-site/doc/example/SearchFieldFocus';
+import SearchFieldProgrammaticSet from '../example/SearchFieldProgrammaticSet';
+import SearchFieldProgrammaticSetSrc from '!raw-loader!../../../../src/terra-dev-site/doc/example/SearchFieldProgrammaticSet';
```

Comment:
Since you're already updating this file you could switch it over to mdx

Reply:
Switched here https://github.com/cerner/terra-core/pull/2800/commits/01caa1861a37d7efb6108958e4396e8fed923d80#diff-03a9a5c821fb01b08d491fecf3df12ccR1